### DISCUSSION TITLE
Setup instructions refer to non-existent file.

### DISCUSSION THREAD

**Discussion by ghost** [chipsalliance/chisel#3801] [General]
The README.md says to follow the setup instruction here:

   https://github.com/chipsalliance/chisel/blob/main/SETUP.md

but the file does not exist.  It makes getting started rather difficult.

**Comment by jackkoenig** [comment]
My apologies, I intended to delete all references to that file in https://github.com/chipsalliance/chisel/pull/3764 but I clearly missed at least one. The new setup instructions are on the website: https://www.chisel-lang.org/docs/installation

I'll fix any lingering links to SETUP.md.

**Reply by jackkoenig** [reply]
The remaining links to `SETUP.md` have been removed.
